### ISSUE TITLE
[Bug] related_name in blog.models.Post made translatable

### ISSUE THREAD

**Issue by restless** [mightylemon/mightylemon#25]
Related_name set for "blog" field in "Post" in blog.models shouldn't be translatable.

The code is:
class Post(models.Model):
    blog = models.ForeignKey(Blog, related_name=_("posts"))

This breaks things as soon as i18n is enabled and "posts" is translated to other language. You get an attribute error when blog.posts is called (for example in blog.views.py in privileged_post_queryset.

The solution is to change the above code to:
class Post(models.Model):
    blog = models.ForeignKey(Blog, related_name="posts")

**Comment by scotu** [comment]
this should be fixed in scotu/mightylemon@37a087f3823fbc02b668154ac97b5720c45b1e8c

Other (little) work has been done in my [locale-it branch](http://github.com/scotu/mightylemon/tree/locale-it) on translatable strings apart from italian translations if you are interested

**Comment by restless** [comment]
Thanks for the answer. I've got one more question to you. How do you deal with locale settings? After I set LANGUAGE_CODE='pl' I have to do some hacks to have archive working. Here is what happens:
1. template filter some_date|date:"b" returns for example "czerw" as month name (it is "jun" in Polish) and it is no longer compatibile with urls resolver expecting (?P<month>[a-z]{3})
2. Post model's get_absolute_url is not affected by locale and it always returns 'jun'. The method get_absolute_url uses strftime and not django.utils.format. The latter returns 'czerw'. Strange but this is how it works for me.
3. I can't just add month_format to generic view because the format is also hardcoded in get_absolute_url, so I also have to modify this method.

I wonder what is your solution for the above issues.

**Comment by scotu** [comment]
I've noticed that too, but I haven't dugg into the problem right now... I'm glad you've done so :) thanks

The best long term solution I see is avoid using the three-letters formatting for the month but go with the number. Doing so the url does not need to be translated and we can have one url for one resource instead of lots of urls for one resource (web crawlers...)

I'm looking forward other proposals...

**Comment by restless** [comment]
Yup. This is what I did. I replaced %b with %m (number of month) in blog.models.Post's get_absulute_url and changed blog.models.urls adding a month_format:'%m' to date_based_dict as well as modfied regexes to expect a number: (?P<month>\d{2}).
